test/flowreg/test.c

```diff
@@ -130,7 +142,10 @@ static int child_main(void)
   mtev_dso_post_init();
 
   mtev_conf_write_log();
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wcast-function-type"
```

Comment:
please provide warning

Reply:
This will be the same fix we settled on for the pool-shift-async test.